releasetasks/templates/source.yml.tmpl

```diff
@@ -72,3 +73,32 @@
             build:
                 platform: none
         #}
+-
+    taskId: "{{ stableSlugId(buildername_signing) }}"
```

Comment:
Hmm, maybe... We would need to pass multiple variables to the template in this case. requires, scopes (may require signing:format:mar, for example), metadata.{name,description}. Still need to think about how to reference this task in the "all deliverables ready" template...